### PR TITLE
Revert "Translation-ja: Add product-owner/ja/03-major-benefits"

### PR DESCRIPTION
**Pull request by ystk**
Reverts InnerSourceCommons/InnerSourceLearningPath#416

The original PR was merged without review. The purpose of this revert PR is for review.

### CODE REVIEW

**Comment by ystk** [comment]
As @rrrutledge mentioned in #416 , I revier #416 first. Then added same document for review. 
Please use this PR, @johtani .

**Review by johtani — CHANGES_REQUESTED** [review]
Thanks @ystk for making new PR.
I left a comment, please take a look.

File: product-owner/ja/03-major-benefits-are-built-into-the-innersource-process-article-ja.asciidoc [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+他のチームが何に取り組んでいるか、現在何を優先しているかがわかれば、チーム間の連携をより効果的に行うことができます。
+オープンプランニングは、チーム間の交渉をより効果的なものにします。
+
+私たちが InnerSource でよく取り組んでいることの1つに、 _オープンドキュメント_ があります。
+これには、プランニングプロセス、標準などの種類のものが含まれます。
+これの重要なポイントは、見つけやすさと、ある程度の中央集権化が行われているという事にあります。そのため、より適切なコラボレーション方法を理解するためのドキュメントがどこにあるか、全ての人が知っています。
+
+それは、あなたの成功に関わる多くのメリットがあります。
+その中にはオープンプランニングやオープンスタンダードがあり、より良いコラボレーションが可能になります。
+しかし、上級管理職が、あなたが経験しているさまざまな道筋や、プロジェクトの過去と今後に関してあなたが何をしているかを見ることができるという信用の側面もあります。
```
[the remark this answers is not in the document]
Comment (ystk) [reply product-owner/ja/03-major-benefits-are-built-into-the-innersource-process-article-ja.asciidoc]: Thanks a lot for your review, @johtani . I make a change to apply your suggestion. 

I am not sure why that label shows Outdated. Maybe I made a mistake..

**Review by johtani — APPROVED** [review]
LGTM!

**Pull request merged**